client/components/theme/test/__snapshots__/index.jsx.snap

```diff
@@ -36,7 +36,9 @@ exports[`Theme rendering with default display buttonContents should match snapsh
       <span
         class="theme__more-button"
       >
-        <button>
+        <button
+          aria-label="More options for theme Twenty Seventeen"
```

Comment:
Should this be translated?

Reply:
This is a test snapshot, I don't think we run any snapshots outside of English. I don't think we need to translate this.